docs/isomorphic-routing.md

```diff
@@ -67,14 +67,12 @@ export function notFoundError( err, context, next ) {
 }
 
 export default function( router ) {
-  router( '/themes/:slug/:section?/:site_id?', details, makeLayout );
-  router( themeNotFound );
+  router( '/themes/:slug/:section?/:site_id?', details, makeLayout, themeNotFound );
 }
 ```
 
-Note that in `notFoundError`, `err` is passed as an argument to `next`. This is how error middleware chains skip regular middlewares. The rendering middleware that is implicitly called on the server after all other middlewares are invoked uses `err.status` to set the HTTP error status.
-
-On the other hand, an error-handling middleware like `themeNotFound` will be called if any other middleware before it calls `next` with an error object:
+Note that you can have multiple error-handling middlewares in your route defintion. When any of the regular middlewares throw an error (or call `next(err)`), only error-handling will be called from that point. This is how error middleware chains skip regular middlewares. The endering middleware that is implicitly called on the server after all other middlewares are invoked uses `err.status` to set the HTTP error status. It will also log an error in the server log, using
```

Comment:
```suggestion
Note that you can have multiple error-handling middlewares in your route defintion. When any of the regular middlewares throw an error (or call `next(err)`), only error-handling will be called from that point. This is how error middleware chains skip regular middlewares. The rendering middleware that is implicitly called on the server after all other middlewares are invoked uses `err.status` to set the HTTP error status. It will also log an error in the server log, using
```